server/routes/dpr/index.ts

```diff
@@ -1,40 +1,56 @@
 import { Request, Response, Router } from 'express'
 import ReportListUtils from '@ministryofjustice/hmpps-digital-prison-reporting-frontend/dpr/components/report-list/utils'
 import config from '../../config'
+import { IncidentReportingApi, ManagementReportDefinition } from '../../data/incidentReportingApi'
+import { Services } from '../../services'
 
 let definitionsRoutesInitialised: boolean = false
 
+async function getIncidentReportingApi(services: Services): Promise<IncidentReportingApi> {
+  const systemToken = await services.hmppsAuthClient.getSystemClientToken()
+  return new IncidentReportingApi(systemToken)
+}
+
+async function populateRoutes(
+  incidentReportingApi: IncidentReportingApi,
+  router: Router,
+): Promise<ManagementReportDefinition[]> {
+  const allDefinitions = await incidentReportingApi.getManagementReportDefinitions()
+  if (definitionsRoutesInitialised === false) {
+    for (const definition of allDefinitions) {
+      for (const variant of definition.variants) {
+        router.get(
+          `/${definition.id}-${variant.id}`,
```

Comment:
out of interest, how come you’ve swapped `/` for `-`?

Reply:
No real reason - think it's nicer